QUARKUS-6752.md

```diff
@@ -0,0 +1,50 @@
+# QUARKUS-6752 Testing and Verification of Semeru CE 21 & 17 on x86_64 (RHEL 8)
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-6752
+
+Upstream issue: https://github.com/quarkusio/quarkus/issues/50570
+
+Semeru JDK is an IBM's release of JDK, using OpenJDK's libraries and IBM's OpenJ9 JVM [[1]](https://developer.ibm.com/languages/semeru-runtimes/).
+We're going to integrate Semeru JDK into quarkus-qe testing infrastructure and executing test suites with it.
+Test suites themselfs should require little to no changes.
+Semeru JDK comes in two editions: Certified edition (CE) and Open Edition (OE).
+Main difference being that OE has open source licence while CE is IBM licensed.
+
+In this instance we will use Semeru JDK 21 in OE or CE (whatever will be more feasible to use). 
+
+## Infrastructure changes
+Semeru JDK will need to be installed on our test executors.
+Semeru JDK is available in RHEL repositories, but only since RHEL 10.
+For that reason, we will be installing it from a package downloaded from [[2]](https://developer.ibm.com/languages/java/semeru-runtimes/downloads/).
+Package can be downloaded and easily installed on RHEL 8.
+
+Semeru JDK will need to be added to jenkins java enum.
+For testing, new jenkins jobs will be created, using just different JDK.
+
+## Scope of testing
+Initially main focus will be on quarkus main, not on supported streams.
```

Comment:
We only support product, you are creating TP for https://issues.redhat.com/browse/QUARKUS-6752 which targets IBQ 27.0.GA. The Quarkus main is nice to have so that we "shift left", but the main focus must be on a IBQ version you target.